dingo/PolytopeSampler.py

```diff
@@ -203,6 +203,7 @@ def generate_steady_states_no_multiphase(
         P = HPolytope(self._A, self._b)
 
         samples = P.generate_samples(method, n, burn_in, thinning, self._parameters["fast_computations"])
+        samples = samples.T
```

Comment:
I think it is more clear to call it by a different name than change the "type" and keep the same name. e.g.
`samples_T = samples.T`

Reply:
Thanks. Done.